### PR TITLE
Command exec on editing the msg

### PR DESCRIPTION
**Pull request by yofukashino**
If an edited msg contains command for the bot then it will exec it

### CODE REVIEW

File: src/listeners/messageUpdate.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	 if (newMessage.webhookID) return; // Check for webhook
+
+ 
+  if (
+    newMessage.member && 
+    newMessage.id === newMessage.member.lastMessageID &&
+    !oldMessage.command
+  ) {
+	 
+    client.emit('message', newMessage);
```
Comment (Allvaa) [inline src/listeners/messageUpdate.js:14]: please dont emit the event

**Review by Allvaa — CHANGES_REQUESTED** [review]

File: src/listeners/messageUpdate.js (hunk 1 above)
Comment (yofukashino) [reply src/listeners/messageUpdate.js:14]: IDK what else to do~ it was easy this way :)

Comment (Allvaa) [reply src/listeners/messageUpdate.js:14]: nah, the event must be triggered as it should be

Comment (yofukashino) [reply src/listeners/messageUpdate.js:14]: IDK how to do that~

**Pull request closed without merging**